src/strategies/orange-reputation-based-voting/index.ts

```diff
@@ -0,0 +1,39 @@
+import fetch from 'cross-fetch';
+
+export const author = 'orange-protocol';
+export const version = '0.1.0';
+
+export async function strategy(
+  space,
+  network,
+  provider,
+  addresses,
+  options,
+  snapshot
```

Comment:
We need to pass snapshot to API and API should make sure it returns data based on the blocknumber else the results would be changing in future